crates/cli/Cargo.toml

```diff
@@ -37,6 +37,10 @@ criterion = "0.3"
 num-format = "0.4.3"
 tempfile = "3.4.0"
 
+[build-dependencies]
+anyhow = "1.0.71"
+wizer = { git = "https://github.com/bytecodealliance/wizer" }
```

Comment:
Can you rebase on main and set the `rev` for this? We realized yesterday that `cargo install` ignores the `Cargo.lock` file so we can get into a state where `cargo install` fails when `main` moves far enough ahead of what's in the lockfile.